Functions.Templates/Templates/EventGridBlobTrigger-CSharp-Isolated-6.x/EventGridBlobTriggerCSharp.cs

```diff
@@ -3,23 +3,22 @@
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 
-namespace Company.Function
+namespace Company.Function;
+
+public class EventGridBlobTriggerCSharp
 {
-    public class EventGridBlobTriggerCSharp
-    {
-        private readonly ILogger<EventGridBlobTriggerCSharp> _logger;
+    private readonly ILogger<EventGridBlobTriggerCSharp> _logger;
 
-        public EventGridBlobTriggerCSharp(ILogger<EventGridBlobTriggerCSharp> logger)
-        {
-            _logger = logger;
-        }
+    public EventGridBlobTriggerCSharp(ILogger<EventGridBlobTriggerCSharp> logger)
+    {
+        _logger = logger;
+    }
 
-        [Function(nameof(EventGridBlobTriggerCSharp))]
-        public async Task Run([BlobTrigger("PathValue/{name}", Source = BlobTriggerSource.EventGrid, Connection = "ConnectionValue")] Stream stream, string name)
-        {
-            using var blobStreamReader = new StreamReader(stream);
-            var content = await blobStreamReader.ReadToEndAsync();
-            _logger.LogInformation($"C# Blob Trigger (using Event Grid) processed blob\n Name: {name} \n Data: {content}");
-        }
+    [Function(nameof(EventGridBlobTriggerCSharp))]
+    public async Task Run([BlobTrigger("PathValue/{name}", Source = BlobTriggerSource.EventGrid, Connection = "ConnectionValue")] Stream stream, string name)
+    {
+        using var blobStreamReader = new StreamReader(stream);
+        var content = await blobStreamReader.ReadToEndAsync();
+        _logger.LogInformation($"C# Blob Trigger (using Event Grid) processed blob\n Name: {name} \n Data: {content}");
```

Comment:
We should probably not encourage bad logging practices in our samples (but not sure if that is in the scope of this PR)